### PR TITLE
Get quarterly cohort numbers from v2 schema when enabled

### PR DESCRIPTION
**Pull request by rsanheim**
**Story card:** [ch5871](https://app.shortcut.com/simpledotorg/story/5871/grab-cohort-data-from-reporting-pipeline)

## Because

We have the cohort numbers in the reporting pipeline but need to grab them and display them for reports

## This addresses

showing reports from the pipeline for cohort

## Test instructions

check out the branch, view cohort reports, cycle between v1 and v2 via query string to compare

### CODE REVIEW

**Comment by rsanheim** [comment]
There are some controller specs that need fixing, but I can handle that Tuesday.  The core services are all in order finally.

File: app/services/reports/region_cache_warmer.rb [hunk 1]
```diff
@@ -62,6 +63,9 @@ def warm_patient_breakdown_caches(batch)
       end
     end
 
+    def warm_cohort_caches(regions)
+    end
```
Comment (ssrihari) [inline app/services/reports/region_cache_warmer.rb:67]: Is this placeholder for upcoming code?

File: app/views/reports/regions/cohort.html.erb [hunk 2]
```diff
@@ -43,7 +43,7 @@
           <div class="col-lg-9 order-lg-2">
             <table class="split-bars">
               <tr>
-                <% if cohort["registered"] > 0 %>
+                <% if cohort["registered"] && cohort["registered"] > 0 %>
```
Comment (ssrihari) [inline app/views/reports/regions/cohort.html.erb:46]: We could add a `.present?` to be more explicit.

**Review by ssrihari — APPROVED** [review]
Excited for this to go in!

**Pull request merged**